script/core/DeployAllo.sol

```diff
@@ -0,0 +1,349 @@
+// SPDX-License-Identifier = MIT
+pragma solidity ^0.8.19;
+
+import {Script, console} from "forge-std/Script.sol";
+import {TransparentUpgradeableProxy} from
+    "openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {ProxyAdmin} from "openzeppelin-contracts/contracts/proxy/transparent/ProxyAdmin.sol";
+
+import {Allo} from "contracts/core/Allo.sol";
+import {DeployBase} from "script/DeployBase.sol";
+
+contract DeployAllo is DeployBase {
+    error NETWORK_NOT_SUPPORTED();
+
+    function _deploy() internal virtual override returns (address _contract, string memory _contractName) {
```

Comment:
We should define a bool that if set to true it will also deploy the registry, because even though we dont need it for production since we are using the existing one, we still need it for testing purposes on local env